app/views/activities/index.html.erb

```diff
@@ -31,7 +31,7 @@
                 <div class="card-descrition">
                   <h3 class="text -title-card"><%= activity.title %></h3>
                   <p class="text -summary"><%= activity.short_description %></p>
-                  <p class="text -meta">1 week ago</p>
+                  <p class="text -meta"><%= time_ago_in_words(activity.created_at) %></p>
```

Comment:
You need to add `ago` after the ruby injection. As the method `time_ago_in_words` just transforms the date into the string with the time (e.g.: "12 minutes" )